### ISSUE TITLE
RedissonDelayedQueue stop transfer timeout elements to destination queue

### ISSUE THREAD

**Issue by lzy-125** [redisson/redisson#5575]
After using for a while, the RedissonDelayedQueue stop work。All added elements are backlogged in the queue，even if many elements have already timeout。 When I restarted the application，all the elements will transfer to the destination queue。
The code is as follows

```
   @PostConstruct
    private void init() {
           StringCodec codec = new StringCodec();
           rBlockingQueue = redissonClient.getBlockingQueue("xxx", codec);
           delayedQueue = redissonClient.getDelayedQueue(rBlockingQueue);
           ScheduledExecutorService executorService = Executors.newScheduledThreadPool(2);
           executorService.scheduleAtFixedRate(this::executeDelayJob, 60, 15, TimeUnit.SECONDS);
    }

    private void executeDelayJob() {
            String xxx = rBlockingQueue.poll();
    }
```

I try to used 
`pubsub numsub redisson_delay_queue_channel:{xxx}`
`redisson_delay_queue_channel:{xxx}`
`(integer) 0`

find that redission client and redis subscriptions are disconnected, it caused RedissonDelayedQueue stop work
My Redisson version is 3.11.0

**Comment by mrniko** [comment]
It can be connected to the issues with topic resubscription fixed in 3.13.1, 3.16.5, 3.19.0. Try 3.19.0 or higher version.

**Comment by mrniko** [comment]
RDelayedQueue successor is [Reliable Queue](https://redisson.pro/docs/data-and-services/queues/#reliable-queue) with delay parameter.
